nuggetsapp/urls.py

```diff
@@ -1,7 +1,10 @@
-from django.conf.urls import url
+from django.conf.urls import url, include
 from . import views
 
 urlpatterns = [
+    url(r'^api/v1/nuggets/(?P<user_id>[0-9]+)/$', views.nuggets_for_user, name='nuggets_for_user'),
```

Comment:
+1 on /user/<user_id>/nuggets and on 2 and 3. 

For nuggets for user, we might want 

4. Also, we need an endpoint for list of review nuggets: /user/<user_id>/review -- the reminder email job will call this to get the list of nuggets to send out. 
5. Register/login user
6. Create/add/update/delete nugget